scripts/uninstall.sh

```diff
@@ -28,6 +15,15 @@ if [ -z "$bash_completion_dir" ]; then
     # BASH_COMPLETION_USER_DIR was not set
     bash_completion_dir="$xdg_home/bash-completion"
 fi
-
 #remove bash completion link
 rm -f "${bash_completion_dir}/completions/$PROGRAM_NAME"
+
+echo "For Bash:"
+echo "- You can remove ~/.local/bin from your path by removing a line in the form \"PATH=\$HOME/.local/bin:\$PATH\" from ~/.bashrc"
```

Comment:
You can wrap the string with single quotes instead of double quotes in order to not having to escape `$` or `"`.

Reply:
Thanks